src/services/userServices.ts

```diff
@@ -16,15 +16,15 @@ export const USER_SERVICES = {
   },
   getAllUsers: async (page: number, limit: number) => {
     try {
-      const res = await apiInstance.get(`${SERVICES.user.get}?page=${page || 1}&limit=${limit || 10}`);
+      const res = await apiInstance.get(`${SERVICES.user.get}?page=${page || 1}&limit=${limit || 1000}`);
       return res.data;
```

Comment:
Please have a proper limit @raja-2604

Reply:
> Please have a proper limit @raja-2604

@harish-flyers as chandran bro mentioned to increase the page limit due to not completed the pagination from the backend and will revert once the updated.